<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodePane</class>
 <widget class="QWidget" name="EncodePane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMainEncodingPane">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxFormEncoding">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutFilePrint">
          <item>
           <widget class="QLabel" name="labelFilePrint">
            <property name="text">
             <string>Empreinte du fichier :</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutFilePrint">
            <item>
             <widget class="QRadioButton" name="radioButtonSHA1">
              <property name="text">
               <string>SHA1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonMD5SUM">
              <property name="text">
               <string>MD5SUM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonNoneFilePrint">
              <property name="text">
               <string>Aucun</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacerFilePrint">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutSendSheetToDB">
          <item>
           <widget class="QCheckBox" name="checkBoxSendSheetToDB">
            <property name="text">
             <string>Envoie de la fiche apr√®s empaquetage du fichier</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerSendSheetToDB">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listViewExport"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxEncodingLog">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>365</width>
             <height>236</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="loggerLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QueueListView" name="listViewProjects"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Encode">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_EncodingButtons">
        <item>
         <widget class="QPushButton" name="pushButton_Encode">
          <property name="text">
           <string>Encoder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Pause">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_Encoding">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueueListView</class>
   <extends>QListView</extends>
   <header>src/View/QueueListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
